src/main/kotlin/uk/co/craigbass/pratura/domain/Product.kt

```diff
@@ -2,4 +2,4 @@ package uk.co.craigbass.pratura.domain
 
 import java.math.BigDecimal
 
-class Product(var sku: String, var price: BigDecimal)
\ No newline at end of file
+class Product(var sku: String, var price: BigDecimal, val name: String)
```

Comment:
I dunno what idiomatic Kotlin looks nice but in Scala it is nice to wrap stringy values that aren't arbitrary user-facing text into a single valued data class (this is about Sku)